aptos-move/vm-genesis/src/lib.rs

```diff
@@ -872,9 +870,8 @@ pub fn test_genesis_module_publishing() {
         NativeGasParameters::zeros(),
         AbstractValueSizeGasParameters::zeros(),
         LATEST_GAS_FEATURE_VERSION,
-        false,
-        true,
         ChainId::test().id(),
+        Features::default(),
```

Comment:
Does the default value of Features (0b00100000) match the existing default values here ?

Reply:
May be not here but the change earlier in this file. We might be lacking direct integration tests that test specific features.